builder/gen-dockerfile/contents/src/gen-files.ts

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+import * as fs from 'fs';
+import * as util from 'util';
+import * as path from 'path';
+
+import { Setup } from './detect-setup';
+import { Reader, Writer, FsView } from './fsview';
+
+async function genFile(writer: Writer, genFiles: Map<string, string>,
+                       name: string, contents: string) {
+  await writer.write(name, contents);
+  genFiles.set(name, contents);
+}
+
+export async function genConfig(config: Setup, appDirWriter: Writer): Promise<Map<string, string>> {
+  const genFiles = new Map();
+  const dataDirReader = new FsView(path.join(__dirname, 'data'));
+
+  // Generate app.yaml
```

Comment:
I thought `app.yaml` still needed to occur so that the user can emulate `gcloud app gen-config` by running the runtime builder Docker image.  If it's not needed, I can remove it.